### DISCUSSION TITLE
custom name save bug

### DISCUSSION THREAD

**Discussion by 2002steef** [mifi/lossless-cut#1252] [Support]
When saving a file under a custom name when said custom name isn't the original name. The extention isn't saved. F.E:

When trying to save a video "GeneralGameClip109", and cutting it down to a highlight, F.E: "LookAtThisFlick", the .mp4 extention isn't shown when selecting the custom name option in the preset. However when saving with the new custom name, there is no extention saved. It saves it as file "LookAtThisFlick", but not as "LookAtThisFlick.mp4", even when having .mp4 extention selected at the output format.

**Comment by mifi** [comment]
Hi. I think you've accidentally removed the `${EXT}` from the template. You can try to reset the template. I see that this is a very common problem, so it's something I want to improve shortly. See https://github.com/mifi/lossless-cut/issues/1083
